src/components/v2/devtronStackManager/devtronStackManager.component.scss

```diff
@@ -0,0 +1,255 @@
+                p {
+                    margin: 0 0 24px;
+                }
+
+                .module-details__features-list-heading {
+                    margin: 0 0 16px;
+                }
+            }
+        }
+    }
+}
+
+.module-details__install-wrapper {
+    .module-details__install-button {
+        width: 100%;
+        height: 36px;
+        
+        &.cta {
+            text-transform: none;
+        }
+
+        .module-details__install-icon {
+            transform: rotate(90deg);
+        }
+
+        .module-details__retry-install-icon {
+            path {
+                fill: none;
+            }
+        }
+    }
+    .module-details__installtion-status {
+        width: 100%;
+        border: 1px solid var(--N200);
+        padding: 16px;
+
+        &.upgrade.status-healthy {
+            position: relative;
+            background-color: white;
+            border: 1px solid var(--N200);
+            overflow: hidden;
+            padding: 16px;
+
+            .module-details__upgrade-success {
+                text-align: center;
+
+                img {
+                    position: absolute;
+                    width: 100%;
+                    bottom: -28px;
+                }
+
+                svg,
+                span {
+                    z-index: 1;
+                }
+            }
+        }
+
+        &.status-installed {
+            background-color: var(--G100);
+            border: 1px solid var(--G200);
+        }
+
+        &.status-installFailed,
+        &.status-timeout,
+        &.status-upgradeFailed,
+        &.status-unknown {
+            background-color: var(--R100);
+            border: 1px solid var(--R200);
+        }
+
+        &.status-installed,
+        &.status-installFailed,
+        &.status-timeout,
+        &.status-upgradeFailed,
+        &.status-unknown,
+        .module-details__installtion-note {
+            padding: 10px 16px;
+        }
+
+        .loader.default-background {
+            justify-content: flex-start;
+        }
+
+        .module-details__installtion-note {
+            margin: 12px 0 0;
+        }
+    }
+
+    .module-details__get-help {
+        width: 100%;
+        height: 124px;
+        border: 1px solid var(--N200);
+        padding: 12px;
+        text-decoration: none;
+
+        a {
+            height: 36px;
+
+            &.module-details__help-guide {
+                svg path {
+                    stroke: var(--B500);
+                }
+            }
+
+            &.module-details__help-chat {
+                svg {
+                    path:first-of-type {
+                        stroke: var(--B500);
+                    }
+
+                    path:not(:first-of-type) {
+                        fill: var(--B500);
+                    }
+                }
+            }
+        }
+    }
+
+    .module-details__update-note {
+        width: 100%;
+        background-color: var(--B100);
```

Comment:
PLease use CSS formulae wherever possible

Reply:
Yes, will try. Some of the things were getting overridden by global CSS so had to use specific selectors.